src/wasm-binary.h

```diff
@@ -424,6 +424,7 @@ enum EncodedType {
   stringview_iter = -0x1f,  // 0x61
   // type forms
   Func = -0x20,   // 0x60
+  Cont = -0x23,   // FIXME(frank-emrich) what do these comments mean?
```

Comment:
That's the signed LEB encoding. Type constructor opcodes are encoded using unsigned LEB. See https://webassembly.github.io/spec/core/binary/types.html

Reply:
I chose `-0x23` arbitrarily. The signed encoding is `5d`, I believe.